x/upgrade/keeper/keeper.go

```diff
@@ -48,6 +49,39 @@ func (k Keeper) SetUpgradeHandler(name string, upgradeHandler types.UpgradeHandl
 	k.upgradeHandlers[name] = upgradeHandler
 }
 
+// SetVersionMap saves a given version map to state
+func (k Keeper) SetVersionMap(ctx sdk.Context, vm module.VersionMap) {
```

Comment:
```suggestion
func (k Keeper) SetModuleVersionMap(ctx sdk.Context, vm module.VersionMap) {
```

I think this is clearer